samples/nrf9160/secure_services/Kconfig

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+source "Kconfig.zephyr"
+
+menu "Secure Service Sample"
+
+config SAMPLE_SERVICE_RNG
+	bool "Enable code that calls the spm_request_random_number()"
+	default y
+
+# NORDIC_SECURITY_BACKEND is not supported on 53 yet
```

Comment:
you mean the `NORDIC_SECURITY_BACKEND` or the cc310 ?

To my knowledge, you can still use `NORDIC_SECURITY_BACKEND` on nRF53 with mbed TLS vanilla backend.

Reply:
This is copied from subsys/spm/Kconfig. @frkv, is this restriction lifted now? I tried building on nrf53 and it did not seem to build.

Reply:
in function `spm_request_random_number':
    /home/oyro/git/ncs/nrf/subsys/spm/secure_services.c:121: undefined reference to `mbedtls_hardware_poll'

Reply:
seems wrong to have that test, would `depends on CC310_ENTROPY` be more correct ?
as that seems to be the foundation for providing `mbedtls_hardware_poll` funtionality.

https://github.com/nrfconnect/sdk-nrfxlib/blob/master/nrf_security/src/mbedtls/CMakeLists.txt#L134-L138

@frkv Maybe nRF Security should provide a config setting that indicates whether `mbedtls_hardware_poll` is available or not.
Direct checks for `! SOC_NRF5340_CPUAPP` seems wrong to me.

Reply:
Yes. This should point to the CC310 entropy driver, which has a config. In time it will point to CC312 entropy driver as well, but this would need to go in in a different PR

It won't build for nRF5340 ATM.